src/plugins/dashboard/public/locator/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export {
+  DASHBOARD_APP_LOCATOR,
+  DashboardAppLocator,
+  DashboardAppLocatorDefinition,
+  DashboardAppLocatorDependencies,
+  DashboardAppLocatorParams,
+} from './app_locator';
+
+export {
+  DASHBOARD_SHARE_LOCATOR,
```

Comment:
is there a possibility to not have a separate share locator but use main apps locator ? i imagine dashboard would have a single locator (possibly multiple for different views but not multiple for a single view) that allows to navigate to it that could either be used by another apps or by share plugin.

Reply:
if the old one is not capable of everything maybe we should deprecate the old one and create new one

Reply:
@ppisljar The issue I ran into was the existing Locator is for Drilldowns, hits a different Dashboard route, and has different parameter requirements... so I found myself passing a flag for view vs create, and making every parameter optional to make the lists work.  So it made sense to split them into two.

Perhaps this should be renamed 'CREATE', though.

Reply:
ok if its for a different route then it makes sense, i agree about the rename

Reply:
I'll do some due diligence on my part... I was moving fast and there may be a way to combine them better than my initial attempt.